index.js

```diff
@@ -19,27 +20,33 @@ module.exports = function (opts) {
 			return;
 		}
 
-		var res;
 		var fileOpts = objectAssign({}, opts);
-
-		try {
-			res = autoprefixer(fileOpts).process(file.contents.toString(), {
+		var processor = postcss()
+			.use(autoprefixer(fileOpts))
+			.process(file.contents.toString(), {
 				map: file.sourceMap ? {annotation: false} : false,
 				from: file.relative,
 				to: file.relative
```

Comment:
looks like these should be the full path now https://github.com/postcss/postcss/blob/master/Runner%20Guidelines.md#21-set-from-and-to-processing-options